packages/calcite-components/src/components/meter/meter.tsx

```diff
@@ -0,0 +1,284 @@
+  @Prop({ reflect: true }) high: number;
+
+  /** Specifies the component's display, where `"single"` displays a single color and `"range"` displays a range of colors which can specify `low`, `high`, `min` or `max` values. */
+  @Prop({ reflect: true }) fillType: "single" | "range" = "range";
+
+  /** Specifies the Unicode numeral system used by the component for localization. */
+  @Prop() numberingSystem: NumberingSystem;
+
+  /** When `true`, number values are displayed with a group separator corresponding to the language and country format. */
+  @Prop({ reflect: true }) groupSeparator = false;
+
+  /** Specifies the appearance style of the component. */
+  @Prop() appearance: Extract<"outline" | "outline-fill" | "solid", Appearance> = "outline-fill";
+
+  /** Determine if the meter should be displayed with a percent based fill, or based on discreet numbers. This will affect displayed labels */
+  @Prop({ reflect: true }) labelType: "percent" | "units" = "percent";
+
+  /** When `labelType` is `"units"` and either `valueLabel` or `rangeLabels` are `true`, displays beside the `value` and/or  `min` values. */
+  @Prop() unitLabel: "";
+
+  /** When `true`, displays the values of `high`, `low`, `min`, and `max`. */
+  @Prop() rangeLabels: boolean;
+
+  /** When `true`, displays the current value. */
+  @Prop() valueLabel: boolean;
+
+  @Watch("min")
+  @Watch("max")
+  @Watch("low")
+  @Watch("high")
+  @Watch("value")
+  handleRangeChange(): void {
+    this.calculateValues();
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Lifecycle
+  //
+  //--------------------------------------------------------------------------
+  connectedCallback(): void {
+    connectLocalized(this);
+    this.calculateValues();
+  }
+
+  disconnectedCallback(): void {
+    disconnectLocalized(this);
+  }
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private State/Props
+  //
+  //--------------------------------------------------------------------------
+
+  private minPercent = 0;
+
+  private maxPercent = 100;
+
+  @State() lowPercent: number;
+
+  @State() highPercent: number;
+
+  @State() currentPercent: number;
+
+  @State() effectiveLocale = "";
+
+  //--------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  //--------------------------------------------------------------------------
+
+  private getMeterKind(): Kind {
+    const { low, high, min, max, value } = this;
+    const lowest = low ? low : min;
+    const highest = high ? high : max;
+    const aboveLowest = value >= lowest;
+    const belowLowest = value < lowest;
+    const aboveHighest = value >= highest;
+    const belowHighest = value < highest;
+
+    if (!value || (!low && belowHighest) || belowLowest) {
+      return "success";
```

Comment:
These are used as CSS classes right? if so should they be in the CSS object?

Reply:
Yeah, I was using `Kind` type as return but will change to string type add to resources 👍